scripts/k8conduit.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+# Connect to a backend service via an app instance
+#
+
+# TODO
+#
+# - usually more than one redis service. Set default and how to override?
+# - test against azure redis service when available
+# - confirm before running if interactive, a flag to run without confirmation?
+#
+
+Help()
+{
+   echo
+   echo "Script to connect to a k8 backing service via an app service"
+   echo
+
+   echo "Syntax:"
+   echo "   k8conduit [-h|i] app-name -- command [args]"
+   echo "      Connect to the default database for app-name"
+   echo
+   echo "or k8conduit [-h|i] -d db-name -k key-vault app-name -- command [args]"
+   echo "      Connect to a specific database from app-name"
+   echo "      Requires a secret containing the DB URL in the specified Azure KV,"
+   echo "      with name {db-name}-database-url"
+   echo
+   echo "options:"
+   echo "   d            database name, required if connecting to a db other than the app default."
+   echo "   i            input file for a restore. Only valid for command psql"
+   echo "   k            key vault that holds the DB URL."
```

Comment:
Maybe add: the following secrets should be exist in the keyvault: ...

Reply:
updated to have more information